packages/mongodb-log-writer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@mongodb-js/mongodb-log-writer",
+  "description": "A library for writing MongoDB logv2 messages",
+  "version": "2.0.0",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/devtools-shared",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/devtools-shared.git"
+  },
+  "keywords": [
+    "mongodb",
+    "log",
+    "logv2"
+  ],
+  "main": "dist/index.js",
+  "exports": {
+    ".": {
+      "require": "./dist/index.js",
+      "import": "./.esm-wrapper.mjs"
+    },
+    "./mongo-log-writer": "./dist/mongo-log-writer.js",
+    "./mongo-log-manager": "./dist/mongo-log-manager.js"
```

Comment:
I think you might also need `types` entries for the individual exports (as well as a top-level `types` property in this package.json), like e.g. https://github.com/mongodb/js-bson/blob/eca63c9f69225270f852f8097d73bb5b7540986f/package.json#L81